src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.dao.impl.MovieSessionDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+
+@Service
+public class MovieSessionServiceImpl implements MovieSessionService {
+    @Inject
+    private MovieSessionDao movieSessionDao;
+
+    @Override
+    public MovieSession add(MovieSession movieSession) {
+        return movieSessionDao.add(movieSession);
+    }
+
+    @Override
+    public MovieSession get(Long id) {
+        return movieSessionDao.get(id).orElseThrow(() ->
+                new RuntimeException("Movie session with id " + id + " not found"));

```

Comment:
Consider using `DataProcessingException` instead of `RuntimeException` for consistency with other methods and to provide more informative error handling.